src/base/bittorrent/session.cpp

```diff
@@ -1809,7 +1809,7 @@ bool Session::addTorrent(const QString &source, const AddTorrentParams &params)
     if (Net::DownloadManager::hasSupportedScheme(source)) {
         LogMsg(tr("Downloading '%1', please wait...", "e.g: Downloading 'xxx.torrent', please wait...").arg(source));
         // Launch downloader
-        Net::DownloadManager::instance()->download(Net::DownloadRequest(source).limit(10485760 /* 10MB */)
+        Net::DownloadManager::instance()->download(Net::DownloadRequest(source).limit(104857600 /* 100MB */)
```

Comment:
I think this is a good time to have a `int MAX_TORRENT_SIZE` constant in global.h

Reply:
Done.